### DISCUSSION TITLE
404 error

### DISCUSSION THREAD

**Discussion by acski** [community/community#36643] [Other Feature Feedback, Questions, & Ideas]
I added a new html page and it showed up fine [.](https://acski.github.io/uploads/lggf1.html) But added another in exactly the same way and it won't show up [[(-](https://acski.github.io/uploads/bcgf1.html)](url) it remains on the 404 despite adding the correct suffix to the url. Tried another page and the same problems. Any idea why this may be happening?

**Comment by yoannchaudet** [comment]
If you can provide us a URL, we could look at it.

**Reply by acski** [reply]
I meant to attach them, sorry, this is the one that works: https://acski.github.io/uploads/lggf1.html

This is one that does not work:  https://acski.github.io/uploads/bcgf1.html

As I said, I uploaded them in exactly the same way.

**Reply by yoannchaudet** [reply]
Alright.

You setup your Pages site to build with GitHub Actions. But I don't see a workflow in your repository. This is probably a change you did not mean. In the settings Pages you can change your Pages source to "Branch" and select your default `main` branch instead, this way each commit will trigger a deployment.

Your last commit did not trigger a Pages deployment because of that. You can see your deployments on this page: https://github.com/acski/uploads/actions/workflows/pages/pages-build-deployment

**Reply by acski** [reply]
Thanks for the reply though I'm still not sure where I'm making the change. I've attached two screenshots where I can see 'branches' referred to. Is it one of those?

![Screenshot 2022-10-20 at 09 06 15](https://user-images.githubusercontent.com/116163776/196892681-fa0f5688-5a4d-448e-baa1-5b73762dd498.png)

![Screenshot 2022-10-20 at 09 05 58](https://user-images.githubusercontent.com/116163776/196892675-302bb462-84cc-4ddf-963f-697a3088bd32.png)

**Reply by radar-blip** [reply]
Your 2nd screenshot, where it says 'Deploy from brach' choose that. Then choose the 'main'.
That is what you want to do.

**Reply by acski** [reply]
I'd actually already tried that but it is still the same. This is the page where I just changed it: https://acski.github.io/uploads/bcgf1.html

**Reply by acski** [reply]
![Screenshot 2022-10-20 at 10 35 32](https://user-images.githubusercontent.com/116163776/196913487-e88f875a-f6a2-451b-91b0-9a81c00278d7.png)

**Reply by acski** [reply]
And the page that does work is set to 'GitHub Actions'.

**Reply by yoannchaudet** [reply]
Without a workflow in your repository you need the "Deploy from branch" option.

I can access your URL just fine: https://acski.github.io/uploads/bcgf1.html

**Reply by acski** [reply]
It does seem to be working now, perhaps there was a delay from when I first changed it. Anyway, that's great, thanks for your help.

**Comment by radar-blip** [comment]
You need an index.html for your site to be visible in GitHub Pages.
Trying adding one, or change the name of one of the existing ones (to index.html).

**Reply by acski** [reply]
But if that's the case I'm not sure why the first page I uploaded worked ok?

So you mean I simply create one html file called index, upload it in the same way, and then that should make the other pages work?

**Reply by acski** [reply]
![Screenshot 2022-10-19 at 21 28 40](https://user-images.githubusercontent.com/116163776/196797448-bc1629e4-412c-4dee-bde3-26f465947c51.png)

Like this?

**Reply by radar-blip** [reply]
Yes, like that. Now just add some content between the opening and closing `<body>` tags.

 `<body>Content of some sort here</body>` and it should work.

**Reply by acski** [reply]
doesn't seem to have made a difference, unless there is a delay before it works

**Reply by radar-blip** [reply]
Have you tried re-deploying the page?
That is in the repositories Settings, under Pages.
Build and Deploy

**Reply by radar-blip** [reply]
Please see @yoannchaudet's answer _above_ as he seems to know what he's talking about. 😑

**Reply by acski** [reply]
thanks for your help anyway, still trying to resolve it!

**Reply by radar-blip** [reply]
Not a problem.

**Comment by ossanna16** [comment]
Hey there @acski and thank you for asking a good question! I'm glad you found our community 🙂
